.github/workflows/ci-code.yml

```diff
@@ -110,27 +121,60 @@ jobs:
         file: ./coverage.xml
         fail_ci_if_error: false  # don't fail job, if coverage upload fails
 
-  verdi:
 
+  tests-presto:
+
+    needs: [check-requirements]
     runs-on: ubuntu-latest
-    timeout-minutes: 15
+    timeout-minutes: 20
 
-    strategy:
-      fail-fast: false
-      matrix:
-        python-version: ['3.9', '3.12']
+    steps:
+    - uses: actions/checkout@v4
 
+    - name: Install graphviz
+      run: sudo apt update && sudo apt install graphviz
+
+    - name: Install aiida-core
+      uses: ./.github/actions/install-aiida-core
+      with:
+        python-version: '3.11'
+
+    - name: Setup SSH on localhost
+      run: .github/workflows/setup_ssh.sh
+
+    - name: Run test suite
+      env:
+        AIIDA_WARN_v3: 0
+      run: pytest -m 'presto'
+
+
+  tests-sqlite:
```

Comment:
>  It is really just adding tests being run with SQLite and RMQ, right? I doubt there would be many situations where this combination is relevant.

Indeed. The idea was to not really test this combination per se, but to increase the code coverage when running with sqlite backend, under the assumption that many of the tests requiring rmq will also exercise the DB backend. Another minor advantage would be that it would ensure that the `not requires_psql` marker keeps working.

I've temporarily enabled code coverage for presto and sqlite runs. The overall coverage for full tests suite is 77.8%, for presto it's 73.1% and 76.5% for sqlite. 

https://github.com/aiidateam/aiida-core/pull/6425#issuecomment-2137793433

I am fine either way. We could also achieve similar outcome by running the nightly rabbitmq tests with sqlite backend. WDYT?

Reply:
I will remove `tests-sqlite` for now since it seems like little coverage gain for a significant amount of resource cost.